doc/ref_arch/kubernetes/chapters/chapter06.md

```diff
@@ -10,9 +10,58 @@
 
 <a name="6.1"></a>
 ## 6.1 Introduction
+Securing Kubernetes requires several layers of security features to provide end to end security for cloud native applications. It is also important to adopt a layered packaging model which support separation of concerns during image build. A fully integrated security testing should be baked into the CI/CD pipeline. Automated security policies should also be used to flag builds with issue.  Image registries must be monitored to automatically block or replace images with known vulnerabilites, while also ensuring policies are used to gate what can be deployed and who can deploy from the registry.
+
+The following functionalities are recommended for securing kubernetes platform;
+- Image Signing
+- Role-base Access Control
+- Secret Managment
+- Kubernetes Cluster Multi-tenancy security
+  - Tenant can be distinct teams or workload types (Develpoment or Production) within an organisation, each with a namespace
+  - Tenant can also be per dedicated kubernetes cluster
+- Integration with other security ecosystem like monitoring and alerting tools
 
 <a name="6.2"></a>
-## 6.2 Heading
+##  6.2 Principles
+The following are core principles to consider when securing cloud native applications;
+
+- Deploy only secure applications and trusted codes
```

Comment:
"Deploy only secure applications and trusted codes" -- we do not have control over what tenants deploy and so this needs to be qualified to "platform services."